README.md

```diff
@@ -25,13 +25,20 @@ The modules are:
 * Fetches configuration from backend & starts collection of measurements
 * Runtime API for applications to start and prolong measurements and metrics
 
+## Prerequisite
+
+Add the following to your environment's global `gradle.properties`
+
+```
+DEFAULT_CONFIG_URL_PREFIX="url to your configuration server"
+DEFAULT_LOCATION_URL_PREFIX="url to your location server"
+```
+
 ## How to build it
 
 ```bash
 $ git submodule init
 & git submodule update
-$ export DEFAULT_CONFIG_URL_PREFIX="url to your configuration server"
-$ export DEFAULT_LOCATION_URL_PREFIX="url to your location server"
 $ ./gradlew Runtime:assemble Plugin:assemble
```

Comment:
Instead of adding another section to the readme i would elaborate on the build command

```diff
-$ export DEFAULT_CONFIG_URL_PREFIX="url to your configuration server"		
-$ export DEFAULT_LOCATION_URL_PREFIX="url to your location server"
-$ ./gradlew Runtime:assemble Plugin:assemble
+ $ ./gradlew Runtime:assemble Plugin:assemble \
+ $  -PDEFAULT_CONFIG_URL_PREFIX="url to your configuration server" \
+ $ -P DEFAULT_LOCATION_URL_PREFIX ="url to your location server"
```

My rationale is: whenever I see a readme that asks me to set something up first I'm more inclined to not even try using that thing.

Reply:
make sense, will update the PR.